apps/api/v2/src/ee/event-types/event-types_2024_04_15/outputs/get-event-type-public.output.ts

```diff
@@ -87,6 +87,14 @@ class BookingField {
   @Type(() => Source)
   @IsOptional()
   sources?: Source[];
+
+  @IsOptional()
+  @IsNumber()
+  maxLength?: number;
+
+  @IsOptional()
+  @IsNumber()
+  minLength?: number;
```

Comment:
same here we probably also need this in packages/platform/types/event-types/event-types_2024_06_14/inputs/booking-fields.input.ts

Reply:
Fixed with faf765d7e894bd9616017adf2c225eba22072177